src/app/layouts/sidenav/sidenav.component.html

```diff
@@ -0,0 +1 @@
+<p>sidenav works!</p>
```

Comment:
Remove this default html tag from angular and replace it with the real layout for the side nav